src/v/kafka/protocol/fetch.h

```diff
@@ -25,6 +25,8 @@
 
 namespace kafka {
 
+static constexpr model::node_id follower_fetching_sentinel(-2);
```

Comment:
comment on this -2?

Reply:
Actually it is not used. In general this is a special value for replica_id that allow reading from non leader node in Kafka (they have it for debug purposes)